src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
```

Comment:
do not change function documentation

Reply:
If I don't change the length of strings, eslint throws the error: "Line 6 exceeds the maximum comment line length of 80"
![image](https://user-images.githubusercontent.com/91499668/209714248-7123287b-bcd7-4663-90a0-b32a1f550b3a.png)